Sources/SwiftGenKit/Parsers/Parser.swift

```diff
@@ -79,3 +79,9 @@ public extension Parser {
     }
   }
 }
+
+public extension Parser {
+  static var allOptions: ParserOptionList {
+    return []
+  }
```

Comment:
Same, `static let allOptions: ParserOptionList = []` if possible

Reply:
See above, it doesn't.

Reply:
😢